tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -732,8 +730,7 @@ Configuration finished
 
 This creates a canonical set of symbolic links to the Cuda libraries on your system.
 Every time you change the Cuda library paths you need to run this step again before
-you invoke the bazel build command. For the cuDNN libraries, use '6.5' for R2, '7.0'
-for R3, and '4.0.4' for R4-RC.
+you invoke the bazel build command. For the cuDNN libraries, use '7.0' for R3, and '4.0.4' for R4-RC.
```

Comment:
This should probably be updated to be 4.0.7 for R4, since it's out of RC and was bumped up to 4.0.7.